pipeline/risteys_pipeline/config.py

```diff
@@ -0,0 +1,20 @@
+import os
```

Comment:
hm, I think maybe a third option is to split the config into `config.ini` and `config.py` like this: 

```
# In config.ini

[finngen]
MINIMAL_PHENOTYPE_DATA_PATH = ...
WIDE_FIRST_EVENTS_DATA_PATH = ...
ENDPOINTS_DATA_PATH = ...

[finregistry]
MINIMAL_PHENOTYPE_DATA_PATH = ...
WIDE_FIRST_EVENTS_DATA_PATH = ...
ENDPOINTS_DATA_PATH = ...

[shared] # if needed
SOME_SHARED_VARIABLE = ...
```

```
# In config.py

from configparser import ConfigParser
from os import getenv

conf_dataset= getenv("CONF_DATASET", "finregistry")

config = ConfigParser()
config.read("config.ini")

MINIMAL_PHENOTYPE_DATA_PATH = config.get(conf_dataset, "MINIMAL_PHENOTYPE_DATA_PATH")
WIDE_FIRST_EVENTS_DATA_PATH = config.get(conf_dataset, "WIDE_FIRST_EVENTS_DATA_PATH")
ENDOPINTS_DATA_PATH = config.get(conf_dataset, "ENDPOINTS_DATA_PATH")
SOME_SHARED_VARIABLE = config.get("shared", "SOME_SHARED_VARIABLE")
```
And then use the configs as before, e.g. 

```
from risteys_pipeline.config import MINIMAL_PHENOTYPE_DATA_PATH
```

What do you think?

Reply:
Ah, that 3rd option looks useful!

My mind is not set on a clear way forward for the config part. I think we need first to figure out how it would look like to run the pipeline for FinnGen and for FinRegistry.
After that it should be clearer.